packages/frontend/src/utils/getRpcUrl.ts

```diff
@@ -1,9 +1,8 @@
-import { L1_NETWORK } from 'src/utils/constants'
+import { L1_NETWORK, Chain } from 'src/utils'
 import { networks } from 'src/config'
-import { Chain } from './constants'
 
 export const getRpcUrl = (network: string) => {
-  return networks[network]?.rpcUrl
+  return networks[network]?.rpcUrl || networks[Chain.Ethereum].rpcUrl
```

Comment:
I think this should throw instead of defaulting to ethereum rpc url. If the network doesn't exist then that's probably an issue that should be handled